controllers/api/user-routes.js

```diff
@@ -116,38 +116,56 @@ router.delete('/:id', withAuth, (req, res) => {
 // login 
 router.post('/login', (req, res) => {
     
```

Comment:
The way this file-change reads is weird, but all I did was put a Try-Catch around it AND a more helpful JSON response.